src/components/header/_macro-options.md

```diff
@@ -23,8 +23,8 @@
 | Name          | Type                    | Required | Description                                                              |
 | ------------- | ----------------------- | -------- | ------------------------------------------------------------------------ |
 | classes       | string                  | false    | Classes to be added to the masthead logo                                 |
-| large         | HTML                    | false    | Any HTML to render an image for example embedded `<svg>` or `<img>`      |
```

Comment:
please update this for the header component as well

Reply:
Already updated